terraform/terraform.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_version = "~> 1.0"
```

Comment:
Can we pin it to the latest available version? Without the squiggly arrow to avoid surprises